src/client/components/UsersInChannelModal.tsx

```diff
@@ -40,45 +42,51 @@ export function UsersInChannelModal({
         <Box>
           <Header>
             <Heading>
-              {channel.org === EVERYONE_ORG_ID ? (
+              {channel.id.startsWith(DM_CHANNEL_PREFIX) ? (
```

Comment:
I've now seen this code like 3 times or so, maybe it is time for a `isChannelADm` utitility function.

Reply:
Yeah, definitely planning to.